crates/solvers-dto/src/auction.rs

```diff
@@ -80,80 +123,201 @@ pub enum SellTokenSource {
 }
 
 #[serde_as]
-#[derive(Debug, Deserialize)]
+#[derive(Debug, Deserialize, ToSchema)]
 #[serde(rename_all = "camelCase")]
 pub struct InteractionData {
+    #[schema(value_type = Address)]
     pub target: H160,
+    #[schema(value_type = TokenAmount)]
     #[serde_as(as = "HexOrDecimalU256")]
     pub value: U256,
+    #[schema(value_type = String, example = "0x01020304")]
     #[serde(with = "bytes_hex")]
     pub call_data: Vec<u8>,
 }
 
-#[derive(Debug, Deserialize)]
+// todo: There is a conflict between solution's SigningScheme which is in
+// camelCase. There is no way to keep 2 object with the same name in the OpenAPI
+// schema. Temporarily renamed the struct. Must be migrated to the camelCase.
+#[derive(Debug, Deserialize, ToSchema)]
 #[serde(rename_all = "lowercase")]
-pub enum SigningScheme {
+pub enum LegacySigningScheme {
```

Comment:
It looks like these two enums are exactly the same. Can we not combine them? Why is one considered more legacy than the other?

Reply:
Because from the driver side we have lowercased variants while from the solvers - camelCase. https://github.com/cowprotocol/services/blob/e4693684a44bfef158351ae4f57da3bcdbd91963/crates/solvers-dto/src/auction.rs#L140